runtime-kernel/linux-kernel-rpi64-lts/autobuild/build

```diff
@@ -0,0 +1,93 @@
+#
+# The Raspberry Pi build script
+# Copied from `linux-kernel' build.
+#
+
+# RPi4 kernel build starts.
+
+abinfo "Building kernel for Raspberry Pi 4."
+
+abinfo "Copying config and setting up env..."
+
+cp -v autobuild/config-rpi4 "$SRCDIR"/.config
+
+. autobuild/build-common/set-var
+
+abinfo "Building kernel..."
+make
+
+abinfo "Stripping symbols..."
+. autobuild/build-common/strip-modules
+
+abinfo "Installing modules..."
+mkdir -p "$PKGDIR"/usr
+make INSTALL_MOD_PATH="$PKGDIR"/usr modules_install
+
+abinfo "Copying kernel..."
+mkdir -p "$PKGDIR"/usr/lib/rpi64/kernel/"$version"/overlays
+install -Dvm755 "$SRCDIR"/arch/arm64/boot/Image "$PKGDIR"/usr/lib/rpi64/kernel/"$version"/kernel8.img
+
+abinfo "Removing the unnecessary firmware in PKGDIR ..."
+rm -rf "$PKGDIR"/usr/lib/firmware
+
+. autobuild/build-common/ext-hdr
+
+export LOCALNAME_RPI4=$LOCALNAME
+unset LOCALNAME
+
+# RPi4 kernel build ends.
+
+# Since `make mrproper` will destroy the installed kernel modules, we need to make a backup then.
+abinfo "Making backup for kernel and kernel modules..."
+tar -cvf rpi4-kernel.tar $PKGDIR
+
```

Comment:
Since we have set the `PKGDIR` to the parent of the kernel tree, you can safely remove the backup and restore process.